### PR TITLE
https://github.com/h5bp/Effeckt.css/issues/309

### PR DESCRIPTION
**Pull request by ShashankaNataraj**
:v: 
1. Added bower.json with jQuery and modernizr dependencies
2. Added a gitignore entry with bower
3. Changed gruntfile copy task to include copying of jQuery and modernizr
4. Changed script inclusion from cdn to bower in layout.hbs

### CODE REVIEW

**Comment by grayghostvisuals** [comment]
looks good

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ dist
 node_modules/
 /tmp/
 .DS_Store
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+bower_components/
```
Comment (arthurvr) [inline .gitignore]: needs a final newline :)

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Effeckt.css",
+  "version": "1.0.0",
```
Comment (arthurvr) [inline bower.json]: We never reached 1.0 :)

**Comment by ShashankaNataraj** [comment]
Hi @arthurvr, In packages.json, Effeckt.css defines version as 1.0.0, so thought of retaining the same in bower.json also :)

What is the preferred version # in bower.json?

**Comment by ShashankaNataraj** [comment]
@arthurvr Hey, any updates? Please answer my questions above and ill be done with this PR!

**Comment by AaronLayton** [comment]
I would set it back to maybe 0.5.0 - there needs to be another issue raised about what needs to be complete before a v1 launch

**Comment by ShashankaNataraj** [comment]
@AaronLayton OK that is fine, can I go ahead and change package.json also with the version number set to .5.0? I currently see 1.0.0 there also!

**Comment by ShashankaNataraj** [comment]
Will open new PR

**Pull request closed without merging**